### ISSUE TITLE
goplay.be: breakage

### ISSUE THREAD

**Issue by RobbeDelarbre** [uBlockOrigin/uAssets#17627]
### Prerequisites

- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable external filter lists, or I have verified that the issue still occurs without enabling external filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://www.goplay.be/video/de-mol/de-mol-s11/de-mol-s11-aflevering-3`

### Category

breakage

### Description

When Ublock is enabled, the video stays on the loading screen but doesn't play. I have the NLD: EasyDutch regional filter list enabled. It worked perfectly a few days ago. All other adblockers are off...

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.48.4
Chromium: 111
filterset (summary): 
  network: 89693
  cosmetic: 41592
  scriptlet: 17704
  html: 0
listset (total-discarded, last-updated): 
  added: 
    NLD-0: 2574-14, 7h.22m
  default: 
    user-filters: 4-2, never
    easylist: 64583-47, 2d.8h.50m
    easyprivacy: 30816-1177, 2d.8h.54m
    plowe-0: 3750-937, 6d.22h.14m
    ublock-abuse: 78-0, 6d.22h.17m
    ublock-badware: 6589-117, 2d.8h.53m
    ublock-filters: 33717-132, 2d.8h.52m
    ublock-privacy: 302-2, 6d.22h.16m
    ublock-quick-fixes: 180-26, 7h.20m
    ublock-unbreak: 1976-47, 2d.8h.51m
    urlhaus-1: 7092-1, 7h.21m
filterset (user): [array of 4 redacted]
trustedset: 
  added: [array of 58 redacted]
switchRuleset: 
  added: [array of 1 redacted]
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 1087 ms (selfie)
  maxAssetCacheWait: 920 ms
popupPanel: 
  blocked: 11
  blockedDetails: 
    goplay.be: 1
    cxense.com: 1
    doubleclick.net: 1
    gemius.pl: 2
    google.com: 1
    googlesyndication.com: 1
    googletagmanager.com: 1
    imasdk.googleapis.com: 1
    onthe.io: 1
    pbstck.com: 1
```
</details>

**Comment by mapx-** [comment]
Even disabling uBO the video does not load / play, maybe because I'm using a vpn

however, try:

`@@||securepubads.g.doubleclick.net/tag/js/gpt.js$script,domain=goplay.be`

If does not work report directly here EasyDutch list:
https://github.com/EasyDutch-uBO/EasyDutch/issues

**Comment by mapx-** [comment]
@BPower0036

**Comment by BPower0036** [comment]
I can not reproduce. It does just works fine. @JohnyP36 can you reproduce?

**Comment by JohnyP36** [comment]
Yes, the rule `@@||imasdk.googleapis.com/js/sdkloader/ima3_dai.js$script,domain=goplay.be` will be implemented.
